integration/hive/hive-openlineage-hook/build.gradle

```diff
@@ -0,0 +1,356 @@
+/*
+/* Copyright 2018-2025 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+import java.util.function.Supplier
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+    id 'pmd'
+    id 'jacoco'
+    id "com.adarshr.test-logger"
+    id 'com.diffplug.spotless'
+    id 'com.github.johnrengelman.shadow'
+}
+
+group = "io.openlineage"
+
+ext {
+    openLineageVersion = '1.24.2'
```

Comment:
This version is quite updated, and probably should be read from .properties file of a root project

Reply:
yup, done